drivers/wifi/esp32/Kconfig.esp32

```diff
@@ -225,4 +225,25 @@ config ESP32_WIFI_SW_COEXIST_ENABLE
 	help
 	  Software controls WiFi/Bluetooth coexistence. Not supported yet.
 
+config ESP32_WIFI_IRAM_OPT
+	bool "WiFi IRAM speed optimization"
+	default y
+	help
+	  Select this option to place frequently called Wi-Fi library functions in IRAM.
+	  When this option is disabled, more than 10Kbytes of IRAM memory will be saved
+	  but Wi-Fi throughput will be reduced.
+
+config ESP32_WIFI_RX_IRAM_OPT
+	bool "WiFi RX IRAM speed optimization"
+	default y
+	help
+	  Select this option to place frequently called Wi-Fi library RX functions in IRAM.
+	  When this option is disabled, more than 17Kbytes of IRAM memory will be saved
+	  but Wi-Fi performance will be reduced.
+
+config ESP32_WIFI_SW_COEXIST_ENABLE
+	bool
+	help
+	  Software controls WiFi/Bluetooth coexistence. Not supported yet.
```

Comment:
This is not related to the PR. Please remove.

Reply:
Yes, fixed, thanks.